src/excmds.ts

```diff
@@ -1879,7 +1879,31 @@ export async function loadaucmds(cmdType: "DocStart" | "DocLoad" | "DocEnd" | "T
     const aucmds = await config.getAsync("autocmds", cmdType)
     const ausites = Object.keys(aucmds)
     const aukeyarr = ausites.filter(e => window.document.location.href.search(e) >= 0)
+    const owntab = await ownTab()
+    const replacements = {
+        TRI_FIRED_MOZ_TABID: owntab.id,
+        TRI_FIRED_TRI_TABINDEX: owntab.index + 1,
+        TRI_FIRED_MOZ_WINID: owntab.windowId,
+        /* TRI_FIRED_TRI_WININDEX: owntab.windowId, */
```

Comment:
I assume you mean tridactyl's win index which I included as a place holder until I can get it working.